Sources/PostgresNIO/Data/PostgresData.swift

```diff
@@ -58,6 +58,10 @@ public struct PostgresData: CustomStringConvertible, CustomDebugStringConvertibl
             var value = value
             value.moveReaderIndex(forwardBy: 1)
             description = String(decoding: value.readableBytesView, as: UTF8.self)
+        case .int4Range:
+            description = try? PostgresRange<Int32>(from: &value, type: type, format: formatCode, context: .default).description
+        case .int8Range:
+            description = try? PostgresRange<Int64>(from: &value, type: type, format: formatCode, context: .default).description
```

Comment:
since we don't want to support this in PostgresData anymore, is this still needed?

Reply:
Removed here, along with two protocol conformances that enabled it: https://github.com/vapor/postgres-nio/pull/330/commits/24c92570c4c7537923ee723b9fb0efd15c7dbdc1